pkg/helpers/workspace_test.go

```diff
@@ -6,10 +6,10 @@ import (
 	"testing"
 )
 
-func TestManagedClusterSetNameForWorkspace(t *testing.T) {
+func TestComputeWorkspaceName(t *testing.T) {
 	workspaceName := "janedoe"
-	name := ManagedClusterSetNameForWorkspace(workspaceName)
+	name := ComputeWorkspaceName(workspaceName)
```

Comment:
Shall we add some more interesting tests now that we're ensuring root:joe:bob and root:joe-bob don't collide?